<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>88</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AudioTransfer Server Settings</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>66</y>
      <width>500</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="DevicesComboBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>400</width>
      <height>44</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="updateDevices">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>0</y>
      <width>100</width>
      <height>22</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string notr="true">Обновить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="confirmDevice">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>22</y>
      <width>100</width>
      <height>22</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string notr="true">Применить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="confirmCode">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>44</y>
      <width>100</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Применить</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="code">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>44</y>
      <width>400</width>
      <height>22</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhHiddenText</set>
    </property>
    <property name="maxLength">
     <number>8</number>
    </property>
    <property name="placeholderText">
     <string>Код доступа</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
